diff --git a/jquery.ui.touch-punch.js b/jquery.ui.touch-punch.js
index 16ce41d1edf13ba003315af035e2a5180b7e964d..9dcf1d3cd2a51c6f72722e68d0ec54c7d10f5f57 100755
--- a/jquery.ui.touch-punch.js
+++ b/jquery.ui.touch-punch.js
@@ -8,7 +8,8 @@
  *  jquery.ui.widget.js
  *  jquery.ui.mouse.js
  */
-(function ($) {
+
+export default function ($) {
 
   // Detect touch support
   $.support.touch = 'ontouchend' in document;
@@ -177,4 +178,4 @@
     _mouseDestroy.call(self);
   };
 
-})(jQuery);
\ No newline at end of file
+}
