diff --git a/dist/type_resolve_helpers.js b/dist/type_resolve_helpers.js
index cae5102d13bccff4e5606af1dde8ac2129cd18e3..344c7383f747b633f2d50fddcf5e25fb22ff4182 100644
--- a/dist/type_resolve_helpers.js
+++ b/dist/type_resolve_helpers.js
@@ -439,7 +439,10 @@ function createTypeLiteral(children, parent) {
         members.push(ts.createPropertySignature(undefined, node.name, opt, t, undefined));
     }
     let node = ts.createTypeLiteralNode(members);
-    if (parent) {
+//    if (parent) {
+// la ligne précédente occasionne du `Cannot read properties of undefined (reading 'names')`
+// sur la ligne suivante, on remplace par
+    if (parent?.prop?.type) {
         const names = parent.prop.type.names;
         if (names.length === 1 && names[0].toLowerCase() === 'array.<object>') {
             node = ts.createArrayTypeNode(node);
